### ISSUE TITLE
Addon boxes coming checked by default

### ISSUE THREAD

**Issue by andersonfaaria** [edubart/otclient#922]
![screenshot_2](https://user-images.githubusercontent.com/8560144/33408735-28f58e12-d55f-11e7-85ed-34a38c73de68.png)
### Before creating an issue, please ensure:
- [x] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/otclient.494/ for support)
- [x] This issue is reproducible without changes to the code in this repository

### Steps to reproduce (include any configuration/script required to reproduce)
1. Set outfit
2. Uncheck addon boxes
3. Click Ok and Set outfit again

### Expected behaviour

When you go to outfit window for the second time it should load your previous configuration.
If you are not using any addon (but you have at least one) then when you go change your outfit the box should come with with your currently outfit with all the addon boxes unchecked.

### Actual behaviour

They always come checked with the addons you can wear. It means if you have both addon 1 and 2, everytime you go to 'Set Outfit' it will popup with both of them checked, even though you're not using any of them.

**Comment by EvulMastah** [comment]
Wasn't it inspired by vanilla client? IIRC it is (or was) the same there.

**Comment by andersonfaaria** [comment]
> Wasn't it inspired by vanilla client? IIRC it is (or was) the same there.

no, otherwise I wouldn't say the expected behaviour is different

**Comment by okk3** [comment]
Hello @andersonfaaria,

I'm not sure about the correct behaviour and I will not look for it as I don't really feel it makes a difference at all, it's just two clicks anyways, nevertheless I run an addonless version.

The fix for you is here:
https://github.com/edubart/otclient/blob/1addf3e1766ca3fe43bdf1114c0655a971123291/modules/game_outfit/outfit.lua#L103 

Set it to false and you should be good for once.

Best Wishes,
Okke
